src/Common/test-runtime/XUnit.Runtime.depproj

```diff
@@ -8,7 +8,7 @@
          This RID value doesn't really matter, since the assets we are copying are not RID specific, so defaulting to Windows here
     -->
     <NugetRuntimeIdentifier>win7-x64</NugetRuntimeIdentifier>
-    <OutputPath>$(RuntimeDir)</OutputPath>
+    <OutputPath>$(RuntimePath)</OutputPath>
```

Comment:
>RuntimePath [](start = 18, length = 11)

What is the difference between `RuntimeDir` and `RuntimePath`? If they are supposed to be the same, can we remove the definition of one of them and only keep one?